packagers/npm-global/prepare.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+mkdir -p ~/.npm-global
+npm config set prefix '~/.npm-global'
+NPM_CONFIG_PREFIX=~/.npm-global
+PATH=~/.npm-global/bin:$PATH
```

Comment:
Why did this become necessary and what does it really do?

Reply:
Because of node > 10, Node.js does not have the correct permissions to access the global modules folder (needs to be sudo/root). It is a known limitation and documented here https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally